contracts/Linklist.sol

```diff
@@ -33,11 +33,15 @@ contract Linklist is
     using EnumerableSet for EnumerableSet.AddressSet;
 
     event Transfer(address indexed from, uint256 indexed characterId, uint256 indexed tokenId);
-    event Burn(address indexed from, uint256 indexed characterId, uint256 indexed tokenId);
+    event Burn(uint256 indexed from, uint256 indexed tokenId);
 
     modifier onlyWeb3Entry() {
         if (msg.sender != Web3Entry) revert ErrCallerNotWeb3Entry();
+        _;
+    }
 
+    modifier onlyExistingToken(uint256 tokenId) {
```

Comment:
Shall we also add ``onlyExistingToken`` modifier on method ``characterOwnerOf`` and ``ownerOf``?

Reply:
Good idea! It is better to explicitly return a revert reason when the token does not exist.